app/scripts/contentscript.js

```diff
@@ -1,3 +1,5 @@
+/*global chrome*/
```

Comment:
I was going to suggest that you use `extensionizer` instead, but that wouldn't work - that doesn't expose the `braveWallet` API.

Maybe we should update `extensionizer` to add that API? :thinking: In the meantime, this seems OK

Reply:
If `braveWallet` is a similar API to the `chrome`/`extension` APIs that are common in browsers, then it would be a good fit. By the name, it sounds more like a provider..

Reply:
`braveWallet` is an _additional_ API. It would need to be added to the list of APIs in `extensionizer`. At the moment, that package only exports the APIs in that list.

It's not a cross-browser API of course, so I wasn't sure whether it was a good fit for that package.